### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader: Status message not announced by screen reader for the content that is updated on the screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9235]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1; #A11yTCS; #SQL Azure Data Studio; #DesktopApp; #Win32; #Benchmark; #A11y_SQLAzureDataStudio; #A11ySev1; #MAS4.1.3;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA(2019.3.1), JAWS(2020.2001.70)
MAS reference: MAS4.1.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Navigate to connection details.
5. Launch NVDA and navigate till connect button and press enter.
6. Check if proper status message is announced by screen reader for all the data that is updated on the screen.

**Actual:**
Status message about data that is updated on the screen is not announced by the screen reader.

**Issue observed in JAWS and NVDA screen reader.**

**Expected:**
Status message should be announced for all the content that is displayed after navigating to any of the controls present on the screen.

**User Impact:**
Screen reader users might not come to know what is the data that is being updated on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**
**MAS 4.1.3**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=b43968d9-ca1a-4396-b266-6e097ea541a9

**Attachment link reference:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-atkulk_microsoft_com/EaMgmxJL6XpOvuhFDMP_BOABkvbQoCBaonMR-_1fFP6qtQ?e=cSapPM

**Comment by alanrenmsft** [comment]
dupe of #9410

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
